### ISSUE TITLE
unknown development team

### ISSUE THREAD

**Issue by TomSwift** [microsoft/react-native-test-app#991]
### What happened?

I've specified my iOS development team in `app.json`, and that correctly sets the team in the generated xcode project for the primary `ReactTestApp` target.

But for the two additional targets `ReactTestAppTests` and `ReactTestAppUITests` the team shows in XCode as "Unknown Name (UBF8T346G9)".

<img width="1196" alt="Screen Shot 2022-07-12 at 1 49 06 PM" src="https://user-images.githubusercontent.com/575821/178592687-df6203f0-4da1-41fe-b2d6-9176026333c3.png">

### Version

1.3.10

### What platforms are you seeing this issue on?

- [ ] Android
- [X] iOS
- [ ] macOS
- [ ] Windows

### System Information

```true
System:
    OS: macOS 12.4
    CPU: (10) arm64 Apple M1 Max
    Memory: 10.96 GB / 64.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.13.2 - ~/.nvm/versions/node/v16.13.2/bin/node
    Yarn: 1.22.17 - ~/.nvm/versions/node/v16.13.2/bin/yarn
    npm: 8.1.2 - ~/.nvm/versions/node/v16.13.2/bin/npm
    Watchman: 2022.03.14.00 - /opt/homebrew/bin/watchman
  Managers:
    CocoaPods: 1.11.2 - /Users/nick/.rbenv/shims/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.2, iOS 15.2, macOS 12.1, tvOS 15.2, watchOS 8.3
    Android SDK:
      API Levels: 28, 30, 31, 32
      Build Tools: 29.0.2, 30.0.2, 30.0.3, 32.0.0, 32.1.0
      System Images: android-28 | Google Play Intel x86 Atom, android-30 | Google Play ARM 64 v8a, android-32 | Google APIs ARM 64 v8a, android-33 | Google Play ARM 64 v8a
      Android NDK: Not Found
  IDEs:
    Android Studio: 2021.2 AI-212.5712.43.2112.8609683
    Xcode: 13.2.1/13C100 - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_322 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: Not Found
    react-native: Not Found
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

### Steps to Reproduce

Load the test app project in XCode.    Navigate to view **Signing & Capability** settings for the `ReactTestAppTests` target.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tido64** [comment]
@TomSwift: Thanks for reporting this. Would you mind testing the fix?

**Comment by tido64** [comment]
FYI: It got published with [1.3.12](https://github.com/microsoft/react-native-test-app/releases/tag/1.3.12).

**Comment by TomSwift** [comment]
> @TomSwift: Thanks for reporting this. Would you mind testing the fix?

Tested - it works!

Thank you for the quick fix.
